### PR TITLE
[DEV-5673] DEFC Action date fix

### PR DESCRIPTION
**Pull request by emily-brents**
**Description:**
Changes the code for DEFC filter to no longer filter awards by action date, only transactions.

**Requirements for PR merge:**

1. [x] Unit & integration tests updated
2. [x] API documentation updated
3. [ ] Necessary PR reviewers:
    - [ ] Backend
4. [N/A] Matview impact assessment completed
5. [x] Frontend impact assessment completed
6. [X] Data validation completed
7. [N/A] Appropriate Operations ticket(s) created
8. [x] Jira Ticket [DEV-5673](https://federal-spending-transparency.atlassian.net/browse/DEV-5673):
    - [x] Link to this Pull-Request
    - [ ] Performance evaluation of affected (API | Script | Download)
    - [ ] Before / After data comparison

**Area for explaining above N/A when needed:**
```
No changes to matviews, not ops tickets needed
```

### CODE REVIEW

**Comment by BrianZito** [comment]
Is there a test file we can add this case to? We don't have time to set up a whole new suite, but this is an issue we would like to be spotting in advance.

**Comment by emily-brents** [comment]
> Is there a test file we can add this case to? We don't have time to set up a whole new suite, but this is an issue we would like to be spotting in advance.

added test to `test_spending_over_time.py`

File: usaspending_api/search/tests/integration/test_spending_over_time.py [hunk 1]
```diff
@@ -957,3 +957,55 @@ def test_failure_with_invalid_group(client, monkeypatch, elasticsearch_transacti
...
+        award_id=99,
+        is_fpds=True,
+        type="A",
+    )
+    mommy.make("awards.TransactionFPDS", transaction_id=99, piid="0001")
+    mommy.make(
+        "awards.TransactionNormalized",
+        id=100,
+        action_date="2020-01-01",
+        federal_action_obligation=10,
```
Comment (BrianZito) [inline usaspending_api/search/tests/integration/test_spending_over_time.py:995]:
```suggestion
        federal_action_obligation=22,
```
Eliminates the small chance the filter gets changed to do the exact reverse, causing the test to pass for the opposite  reason.

Comment (emily-brents) [reply usaspending_api/search/tests/integration/test_spending_over_time.py]: yeah sure

**Pull request merged**